avm-transpiler/Earthfile

```diff
@@ -1,27 +1,26 @@
 VERSION 0.8
-IMPORT ../noir AS noir
 
 source:
-  # we rely on noir source, which this image has
-  FROM noir+nargo
+    # we rely on noir source, which this image has
+    FROM ../noir+nargo
 
-  # move noir contents to /usr/src/noir
-  RUN mv /usr/src /noir && mkdir /usr/src && mv /noir /usr/src
-  # work in avm-transpiler
-  WORKDIR /usr/src/avm-transpiler
+    # move noir contents to /usr/src/noir
+    RUN mv /usr/src /noir && mkdir /usr/src && mv /noir /usr/src
+    # work in avm-transpiler
+    WORKDIR /usr/src/avm-transpiler
 
-  COPY --dir scripts src Cargo.lock Cargo.toml rust-toolchain.toml .
+    COPY --dir scripts src Cargo.lock Cargo.toml rust-toolchain.toml .
 
-  # for debugging rebuilds
-  RUN echo CONTENT HASH $(find . -type f -exec sha256sum {} ';' | sort | sha256sum | awk '{print $1}') | tee .content-hash
+    # for debugging rebuilds
+    RUN echo CONTENT HASH $(find . -type f -exec sha256sum {} ';' | sort | sha256sum | awk '{print $1}') | tee .content-hash
 
 build:
     FROM +source
     RUN ./scripts/bootstrap_native.sh
     SAVE ARTIFACT target/release/avm-transpiler avm-transpiler
+    SAVE ARTIFACT scripts/compile_then_transpile.sh
```

Comment:
Needed by `aztec-nargo+run`.... Feels like I shouldn't have to do this just to get the script copied there.